internal/view/dir_remote.go

```diff
@@ -0,0 +1,255 @@
+	if os == windowsOS {
+		// FIXME implement
+		// args = append(args, "--", powerShell)
+	}
+
+	// TODO make configurable
+	args = append(args, "--", "ls", "-1aF", "--color=never")
+	if dir != "" {
+		args = append(args, "--", dir)
+	}
+
+	res, err := runKu(a, shellOpts{args: args})
+	if err != nil {
+		a.Flash().Errf("Shell exec '%s' failed: %s", strings.Join(args, " "), err)
+	}
+
+	return res, err
+}
+
+func (d *DirRemote) gotoCmd(evt *tcell.EventKey) *tcell.EventKey {
+	if d.GetTable().CmdBuff().IsActive() {
+		return d.GetTable().activateCmd(evt)
+	}
+
+	sel := d.GetTable().GetSelectedItem()
+	if sel == "" {
+		return evt
+	}
+	if !strings.HasSuffix(sel, "/") { // file
+		return d.transferCmd(evt)
+	}
+	dir := strings.TrimSuffix(sel, "/")
+
+	str, err := listRemoteFiles(d.App(), d.fqn, d.co, d.os, dir)
+	if err != nil {
+		//d.App().Flash().Err(err)
+		return evt
+	}
+
+	/*
+		v := NewDirRemote(d.fqn, d.co, d.os, dir, str)
+		if err := d.App().inject(v, false); err != nil {
+			d.App().Flash().Err(err)
+		}
+	*/
+	d.text = str
+	d.dir = dir
+	d.Start()
+
+	return evt
+}
+
+func makeTransferAck(a *App, path string) func(args dialog.TransferArgs) bool {
+	return func(args dialog.TransferArgs) bool {
+		local := args.To
+		if !args.Download {
+			local = args.From
+		}
+		if _, err := os.Stat(local); !args.Download && errors.Is(err, fs.ErrNotExist) {
+			a.Flash().Err(err)
+			return false
+		}
+
+		opts := make([]string, 0, 10)
+		opts = append(opts, "cp")
+		opts = append(opts, strings.TrimSpace(args.From))
+		opts = append(opts, strings.TrimSpace(args.To))
+		opts = append(opts, fmt.Sprintf("--no-preserve=%t", args.NoPreserve))
+		opts = append(opts, fmt.Sprintf("--retries=%d", args.Retries))
+		if args.CO != "" {
+			opts = append(opts, "-c="+args.CO)
+		}
+		opts = append(opts, fmt.Sprintf("--retries=%d", args.Retries))
+
+		cliOpts := shellOpts{
+			background: true,
+			args:       opts,
+		}
+		op := trUpload
+		if args.Download {
+			op = trDownload
+		}
+
+		fqn := path + ":" + args.CO
+		if err := runK(a, cliOpts); err != nil {
+			a.cowCmd(err.Error())
+		} else {
+			a.Flash().Infof("%s successful on %s!", op, fqn)
+		}
+		return true
+	}
+}
+
+func (d *DirRemote) transferCmd(evt *tcell.EventKey) *tcell.EventKey {
+	path := d.fqn
+	ns, n := client.Namespaced(path)
+	file := d.GetTable().GetSelectedItem()
+	pod, err := fetchPod(d.App().factory, path)
+	if err != nil {
+		d.App().Flash().Err(err)
+		return nil
+	}
+
+	opts := dialog.TransferDialogOpts{
+		Title:      "Transfer",
+		Containers: fetchContainers(pod.ObjectMeta, pod.Spec, false),
+		Message:    "Download Files",
+		Pod:        fmt.Sprintf("%s/%s:%s", ns, n, file),
+		File:       filepath.Base(file),
+		Ack:        makeTransferAck(d.App(), path),
+		Download:   true,
+		Retries:    defaultTxRetries,
+		Cancel:     func() {},
+	}
+	dialog.ShowUploads(d.App().Styles.Dialog(), d.App().Content.Pages, opts)
```

Comment:
I think we should have a base transfer dialog for this. Based on the context aka loca/remote the Download toggle does not mean much anymore and we already know what kind of action should take place ie remote->download local->upload.  
Also not keen on the view titles ie `Dirlocal` or `Dirremote` perhaps something like `Local File System` and `Remote File System` could be more comprehensible??